styles/blocks/staff-panel/person.css

```diff
@@ -0,0 +1,35 @@
+.person__photo {
+    float: left;
```

Comment:
Since its container lacks `clearfix`, it doesn't scale to wrap its children size. Please make sure to add `clearfix` when using float or opt for inline-blocks, flex, etc. In this case it seems to me that there's no need for the image to float in the text.